src/commcare_cloud/commands/deploy/commcare.py

```diff
@@ -44,24 +45,27 @@ def deploy_commcare(environment, args, unknown_args):
         start_time=datetime.utcnow()
     )
 
-    record_deploy_start(environment, context)
+    should_record = not (args.skip_record or args.private)
+    if should_record:
+        record_deploy_start(environment, context)
 
     code_version = context.diff.deploy_commit if not args.resume else ''
     ansible_args = ["-e", f"code_version={code_version}"]
-    if getattr(args, "preindex_views", False):
-        fab_command = "preindex_views"
-    elif args.setup_release:
-        if args.limit:
-            if args.limit == 'django_manage':
-                fab_command = "setup_limited_release"
-            else:
-                exit("--limit only supports 'django_manage' at this time")
-        else:
-            fab_command = "setup_release"
+    if args.private and not args.keep_days:
+        args.keep_days = 1
+    if args.keep_days:
+        until = (datetime.utcnow() + timedelta(days=args.keep_days)).strftime(DATE_FMT)
+        ansible_args.extend(["-e", f"keep_until={until}"])
+    if args.private:
```

Comment:
Just to confirm my understanding, if no `--limit` arg is provided this will create a release on all machines right? What do you think about defaulting the limit to 'django_manage' which is the common use case. If you want the release on all machines you must provide `--limit=[all|*]` (https://docs.ansible.com/ansible/latest/inventory_guide/intro_patterns.html#common-patterns)

Reply:
I like that idea.

Reply:
https://github.com/dimagi/commcare-cloud/pull/5820/commits/c8ce50e706b38a4be35147c8cbce44a5b132ff40